source/plugins/ruby/out_named_pipe.rb

```diff
@@ -0,0 +1,92 @@
+require 'fluent/plugin/output'
+
+$pipe_handle = nil
```

Comment:
why this being defined outside of the module? and what's this signifies?

Reply:
Just defining as a global var. I am working on creating a class variable instead so CONTAINER_INVENTORY and LINUX_PERF have their own pipe handle. Will update it then